source/package.json

```diff
@@ -1,50 +1,31 @@
 {
-  "name": "subtheme",
-  "description": "Subtheme file for us in KIT-related projects",
+  "name": "unitedrentals",
+  "description": "UR: Drupal 8 KIT common theme support",
```

Comment:
Probably change the name and description back to something KIT related.

Reply:
good catch! sorry that got moved in